api-reference/_hidden/dxDataGridColumn/calculateGroupValue.md

```diff
@@ -4,16 +4,170 @@ type: String | function(rowData)
 ---
 ---
 ##### shortDescription
-Specifies a field name or a function that returns a field name or a value to be used for grouping column cells.
+A field name or a function that returns a field name or value by which to group cells. This value is also used to sort **DataGrid**'s groups.
 
 ##### param(rowData): Object
 The current row's data.
 
 ##### return: any
-The calculated value to be used for grouping.
+The calculated value to be used for grouping and group sorting.
```

Comment:
```suggestion
The calculated value used to group and sort columns.
```

Reply:
Why "group sorting"?

Reply:
"group sorting" because the widget sorts groups, not grouped rows. A group is a set or rows.
In this picture the groups are circled:
![image](https://user-images.githubusercontent.com/66619750/100236427-b53bb900-2f3e-11eb-8038-658808963e0f.png)

Can the sentence be "The calculated value by which to create and sort groups."?

Reply:
The calculated value used to create and sort groups.